lib/azure/signature.rb

```diff
@@ -2,13 +2,14 @@
 require 'openssl'
 require 'base64'
 require 'time'
+require 'active_support/core_ext/hash/conversions'
 
 # The Azure module serves as a namespace.
 module Azure
   # The Signature class encapsulates an canonicalized resource string.
   class Signature
     # The version of the azure-signature library.
-    VERSION = '0.2.3'
+    VERSION = '0.3.0'
```

Comment:
https://github.com/ManageIQ/azure-signature/pull/4/files#diff-78bc9c86a5405fa5f8eafe2014559ddbR5 why do we have to write the version in two places in the fist place? What if they are out of sync by accident?